experimental/sdk/webpubsub-socketio-extension/examples/chat-with-negotiate/index.js

```diff
@@ -5,6 +5,7 @@ const { AzureCliCredential } = require("@azure/identity");
 const express = require('express');
 const app = express();
 const path = require('path');
+const { getNegotiateHttpMiddleware } = require("@azure/web-pubsub-socket.io/SIO/components/negotiate");
```

Comment:
why not export at root

Reply:
It's generated by a VS Code feature. Updated